--- I18N_UnicodeString-0.3.0/usr/share/pear/I18N/UnicodeString.php~	2012-03-29 10:21:28.000000000 +0300
+++ I18N_UnicodeString-0.3.0/usr/share/pear/I18N/UnicodeString.php	2012-03-29 10:23:32.223206875 +0300
@@ -41,7 +41,7 @@
 * @filesource
 */
 
-require_once 'I18N/UnicodeString/Exception.php';
+require_once 'I18N/Unicode/Exception.php';
 
 /**
 * Class provides a way to use and manipulate multibyte strings in PHP
